android/config.gradle

```diff
@@ -34,18 +34,19 @@ if (pspdfkitPassword == null || pspdfkitPassword == '') {
     throw new GradleException("PSPDFKit password was not specified. The password is required to download the PSPDFKit AAR file into your project. Please specify as pspdfkit.password inside the local.properties file.")
 }
 
+ext.pspdfkitVersion = localProperties.getProperty('pspdfkit.version')
+if (pspdfkitVersion == null || pspdfkitVersion == '') {
+    ext.pspdfkitVersion = '5.5.0'
+}
+
 def pspdfkitIsDemo = localProperties.getProperty('pspdfkit.demo')
 if (pspdfkitIsDemo == null) {
```

Comment:
Do we need to simplify this block as well? From what I gathered there will be no 'pspdfkit.demo' property anymore

Reply:
We need to keep this, so we can guarantee the compatibility for the older versions (<`5.5.0`) but you are right: from now on the only real check will be on the `useUnifiedBinaries`